bentoml/server/bento_api_server.py

```diff
@@ -59,12 +62,8 @@ def _request_to_json(req):
     """
     if req.content_type == "application/json":
         return req.get_json()
-    elif "image" in req.content_type:
-        return {"data": "dont handle"}
-    elif "video" in req.content_type:
-        return {"data": "dont handle"}
 
-    return {"data": req.get_data().decode("utf-8")}
+    return {"data": "ignore content type other than JSON."}
```

Comment:
just `return {}`? 

for a image handler api server, we don't want to pollute the logs with this message `{"data": "ignore content type other than JSON."}` right?

Reply:
Yes, since we have output the file paths, this won't be needed.